modules/apps/josh-lapurd-theme/src/js/main.js

```diff
@@ -0,0 +1,65 @@
+AUI().ready(
+	'liferay-sign-in-modal',
+	'scroll-transparency',
+	function(A) {
+		A.on(
+			'scroll',
+			function() {
+				var banner = AUI.$('#banner');
+				var buttonBtt = AUI.$('.btn-back-to-top');
+				var scrollY = window.pageYOffset;
+
+				if (scrollY > 350 && banner.hasClass('banner-unscrolled')) {
+					banner.toggleClass('banner-scrolled banner-unscrolled');
+					buttonBtt.fadeIn();
+				}
+					else if (scrollY <= 350 && banner.hasClass('banner-scrolled')) {
+						banner.toggleClass('banner-scrolled banner-unscrolled');
+						buttonBtt.fadeOut();
+					}
+			}
+		);
+
+		var signIn = A.one('#signIn');
+
+		if (signIn && signIn.getData('redirect') !== 'true') {
+			signIn.plug(Liferay.SignInModal);
+		}
+	}
+);
+
+function toggleOpenClose() {
+	var currTarget = AUI.$(this);
+	var currTarSib = currTarget.siblings('ul.child-menu');
```

Comment:
Rename variable to ```currentTargetSibling``` or ```childMenu``` so it is clear what we are getting with this JS variable.